### ISSUE TITLE
Player drop functionality

### ISSUE THREAD

**Issue by MenosGrandes** [pupil-labs/pupil#1796]
I'm checked out onto v1.21 and after starting the player I cannot run any of the recording that I have made, by the drop of the folder onto window.

 I don't have any logs from on_drop function in launchables/player. So it seems that the     `glfw.glfwSetDropCallback(window,·on_drop)` does not work.

It was tested on 3 computers so I have no idea what's wrong.

**Comment by papr** [comment]
Hi @MenosGrandes Does the text in the drop window change after dropping a recording into it? If yes, what does it say? Also, on which operating systems have you tested this? You can find log messages in the Home Directory > pupil_player_settings > player.log file. It would be helpful to us if you could share this file after attempting to drop a recording onto the window.

**Comment by MenosGrandes** [comment]
Hi @papr  there is no info changed on player window. 

`uname -a`

`Linux pupil-OMEN-by-HP-Laptop-15-ce0xx 4.15.0-65-generic #74-Ubuntu SMP Tue Sep 17 17:06:04 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux`

`
pupil@pupil-OMEN-by-HP-Laptop-15-ce0xx / $ cat ./home/pupil/Desktop/pupillabupgrades/player_settings/player.log
2020-01-28 12:34:32,450 - MainProcess - [DEBUG] root: Unknown command-line arguments: []
2020-01-28 12:34:32,450 - MainProcess - [DEBUG] os_utils: Disabling idle sleep not supported on this OS version.
2020-01-28 12:34:32,840 - player - [INFO] video_capture: Install pyrealsense to use the Intel RealSense backend
2020-01-28 12:34:32,841 - player - [INFO] video_capture: Install pyrealsense2 to use the Intel RealSense backend for D400 series cameras
2020-01-28 12:34:32,894 - player - [INFO] launchables.player: Session setting are from a  different version of this app. I will not use those
`

**Comment by pfaion** [comment]
@MenosGrandes can you share the content of either `info.csv` or `info.player.json` (whichever is present) in the folder of the recording?

**Comment by MenosGrandes** [comment]
info.player.json:
`{
    "duration_s": 17.694983270950615,
    "meta_version": "2.0",
    "min_player_version": "1.16",
    "recording_name": "2020_01_28",
    "recording_software_name": "Pupil Capture",
    "recording_software_version": "1.10.1272",
    "recording_uuid": "b9cd0011-b45f-45fb-9080-6a8026804f52",
    "start_time_synced_s": 8371358.510615516,
    "start_time_system_s": 1580201627.8815906,
    "system_info": "User: pupil, Platform: Linux, Machine: pupil-OMEN-by-HP-Laptop-15-ce0xx, Release: 4.15.0-65-generic, Version: #74-Ubuntu SMP Tue Sep 17 17:06:04 UTC 2019"
}`
The recording was done on old softwere commit:
27f9153effd04cc160666416386e83989509a677

I haven't merget the newest ones. But the same error is while I'm using newest version of pupilLab.

**Comment by pfaion** [comment]
This is indeed weird. Can you download the bundled version of v1.21 and give that a try? 
https://github.com/pupil-labs/pupil/releases/tag/v1.21#user-content-downloads

If the bundled version also fails to open we'll have to take a deeper look at the recording.
Otherwise there might be something wrong with your local setup. Did you update all dependencies of Pupil when checking out the new version? Whenever dependencies change we note that in the release notes. If you jumped multiple versions at once you should take a look at the release notes in between to see if we bumped up any dependency versions.

**Comment by papr** [comment]
@MenosGrandes Is there any update from your side? Were you able to solve the issue?

**Comment by MenosGrandes** [comment]
I'm sorry for so late comment, but I had to finish some projects...
I have found out that I can run player from terminal, so I did not investigated why the drag/drop functionality does not work.

BUT! I have at least 4 computers with different OS (Linux based) that have the same problem, using the  PupilLab v1.21.
In next week I will try to update my forked repo to newest PupilLab version and we will find out what will happend.
